src/Runner.Worker/Handlers/ContainerActionHandler.cs

```diff
@@ -232,5 +229,20 @@ public async Task RunAsync(ActionRunStage stage)
             }
 #endif
         }
+
+        private bool IsDockerFile(string image)
+        {
+
+            var imagePathSplit = Data.Image.Split('/');
+            var imageWithoutPath = imagePathSplit[imagePathSplit.Length - 1];
+            if (imageWithoutPath.StartsWith("Dockerfile") || imageWithoutPath.StartsWith("dockerfile") || imageWithoutPath.EndsWith("Dockerfile") || imageWithoutPath.EndsWith("dockerfile"))
```

Comment:
Do we need the `EndsWith` conditions here? It made sense for a full path, but now this would match 'foo-bar.Dockerfile', which is not what we want I think.

Reply:
I left it because of the comment at the end of the issue:

> prefix-naming.Dockerfile, which is not prohibited by Docker and works in GitHub Actions

So I am guessing that people are using it that way